src/applications/mhv-secure-messaging/tests/e2e/pages/GeneralFunctionsPage.js

```diff
@@ -81,6 +81,17 @@ class GeneralFunctionsPage {
       .parent(`p`)
       .should(`include.text`, `End: ${this.getDateFormat(endDate)}`);
   };
+
+  getRandomDateWithinLastNumberOfMonths = number => {
+    const now = new Date();
+    const threeMonthsAgo = new Date();
+    threeMonthsAgo.setMonth(now.getMonth() - number);
```

Comment:
for clear readability, I may suggest restructure naming of variables. Specifically, `threeMonthsAgo` may not necessarily be 3 months ago as that is determined by `number` argument. So it may be confusing. Have you considered using `date-fns` library for easier manipulation with dates?

Reply:
agree, just forgot to rename the variable as I started with 3 month date )
nope, but I will next time.

Reply:
updated